package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gate-teamware",
-  "version": "0.1.4",
+  "version": "0.2.0\n",
```

Comment:
Oops looks like this new line character might have been added by the versioning script?

Reply:
Yes looks like a `.strip()` just needed to be added to the read version string. Will put that in the next commit.